src/algebra/group_power/lemmas.lean

```diff
@@ -366,24 +366,42 @@ end linear_ordered_semiring
 
 section linear_ordered_ring
 
-variables [linear_ordered_ring R]
+variables [linear_ordered_ring R] {a : R} {n : ℕ}
 
 @[simp] lemma abs_pow (a : R) (n : ℕ) : abs (a ^ n) = abs a ^ n :=
 abs_hom.to_monoid_hom.map_pow a n
 
-@[simp] theorem pow_bit1_neg_iff {a : R} {n : ℕ} : a ^ bit1 n < 0 ↔ a < 0 :=
+@[simp] theorem pow_bit1_neg_iff : a ^ bit1 n < 0 ↔ a < 0 :=
 ⟨λ h, not_le.1 $ λ h', not_le.2 h $ pow_nonneg h' _,
   λ h, mul_neg_of_neg_of_pos h (pow_bit0_pos h.ne _)⟩
 
-@[simp] theorem pow_bit1_nonneg_iff {a : R} {n : ℕ} : 0 ≤ a ^ bit1 n ↔ 0 ≤ a :=
+@[simp] theorem pow_bit1_nonneg_iff : 0 ≤ a ^ bit1 n ↔ 0 ≤ a :=
 le_iff_le_iff_lt_iff_lt.2 pow_bit1_neg_iff
 
-@[simp] theorem pow_bit1_nonpos_iff {a : R} {n : ℕ} : a ^ bit1 n ≤ 0 ↔ a ≤ 0 :=
+@[simp] theorem pow_bit1_nonpos_iff : a ^ bit1 n ≤ 0 ↔ a ≤ 0 :=
 by simp only [le_iff_lt_or_eq, pow_bit1_neg_iff, pow_eq_zero_iff (bit1_pos (zero_le n))]
 
-@[simp] theorem pow_bit1_pos_iff {a : R} {n : ℕ} : 0 < a ^ bit1 n ↔ 0 < a :=
+@[simp] theorem pow_bit1_pos_iff : 0 < a ^ bit1 n ↔ 0 < a :=
 lt_iff_lt_of_le_iff_le pow_bit1_nonpos_iff
 
+theorem even_pow_nonneg (a : R) (hn : even n) : 0 ≤ a ^ n :=
+by cases hn with k hk; simpa only [hk, two_mul] using pow_bit0_nonneg a k
+
+theorem even_pow_pos (ha : a ≠ 0) (hn : even n) : 0 < a ^ n :=
+by cases hn with k hk; simpa only [hk, two_mul] using pow_bit0_pos ha k
```

Comment:
Shouldn't these be `pow_even_nonneg`? Using `pow_bit0_pos` as a template, statements about the exponent go after the word `pow` not before.

Reply:
Indeed, you're right, I should have looked more closely at the naming.

Reply:
@eric-wieser @robertylewis Fixed!